styleguide/source/_patterns/02-molecules/tool-drawer.twig

```diff
@@ -0,0 +1,7 @@
+<div class="joe__tool-drawer">
+  <div id="{{ tool_drawer.id }}" class="joe__tool-drawer__content">
+   {{ tool_drawer.content }} 
+  </div>
+  {% set button = tool_drawer.button %}
+  {% include "@atoms/button.twig" %}
+</div>
```

Comment:
It doesn't  look the js is being pulled in on the pattern page so it isn't actually showing the citation working. And there's no way to see the content. I'm not sure what the work around is for this in pattern lab (because I understand that realistically that would link in the footer), however we may want to have the no-js fallback to show all the content (in case users have javascript turned off in their browser).

Reply:
I don't actually understand why the script isn't working on this page (since it works on article-tools molecule).

I'll look into adding a no js detection with fallbacks. Any ideas on how to move forward with that? Based on a cursory glance, there is no real reliable way.